deploy/clowdapp.yaml

```diff
@@ -268,6 +268,27 @@ objects:
           requests:
             cpu: 100m
             memory: 64Mi
+    - name: org-id-populator
+      podSpec:
+        name: org-id-populator
+        restartPolicy: OnFailure
+        image: quay.io/dehort/tenant-utils:5228ba1e3680
+        command:
+          - ./org-id-column-populator
+          - -C
+          - -a
+          - account
+          - -o
+          - org_id
+          - -t
+          - runs
+        resources:
+          limits:
+            cpu: 300m
+            memory: 1Gi
+          requests:
+            cpu: 50m
+            memory: 512Mi
```

Comment:
ClowdJobInvocation will also need to be defined for this to run. https://consoledot.pages.redhat.com/clowder/dev/usage/jobs.html

Reply:
Got it! I added the CJI and tested it in ephemeral and it seems to be working correctly.